src/hapi_schema/db_admin2.py

```diff
@@ -72,3 +72,23 @@ class DBAdmin2(Base):
         ),
     ),
 )
+
+
+class DBadmin2_vat(Base):
```

Comment:
Is there a reason why you don't use camel case for the class name?

Reply:
I was generating class names from text string used to name the table, I've manually edited these.

You should be grateful I don't use FORTRAN-77 SHOUTY CASE! ;-)